frame/democracy/src/benchmarking.rs

```diff
@@ -467,6 +471,36 @@ benchmarks! {
 		}
 	}
 
+	on_initialize_no_launch_no_maturing {
```

Comment:
how is this different from all the other `on_initialize_xxx` benches? would probably be helpful to write it down here.

Reply:
yes on_initialize_external do benchmark the launch of an external referendum and the maturing of R referendums.
and on_initialize_public does same with public referendum.

I agree doc should improve here, I'll try to do in my weight PR